### ISSUE TITLE
Many miscellaneous wallet text changes needed

### ISSUE THREAD

**Issue by VeilDango** [Veil-Project/veil#56]
https://imgur.com/a/cIJkrlK

@StrontiumZ38 

Furszy told me you would be making the changes. Going through list of changes you made, it was mostly changing Bitcoin to Veil. Here are many more. Each item or successive items that are text changes are prefaced with "Text changes" in this imgur album.

Reproducibility:

I can reliably reproduce the issue. Refer to screenshot.

Version used:

https://github.com/Veil-Project/veil/releases/download/testnet-1/veil-0.0.9-win64-setup-unsigned.exe

Hardware and OS:

6700k Intel CPU
Pro 970 SSD
32GB RAM
Windows 10 Pro

Debug.log:

https://cdn.discordapp.com/attachments/507024857847037962/527898028942688256/debug.log

**Comment by StrontiumZ38** [comment]
Many of these have been fixed to the best of my ability to do so with a recent PR. Much of the text is actually just missing as it's incorrectly referenced, and exists in the source. I've created an issue for those items, which are within the FAQ section.

Other items in your album are either already fixed, or are being worked on, and I have your album open to ensure they all get looked at. With that in mind, I'd say this can be closed now.
